apps/dashboard/src/app/(dashboard)/(chain)/[chain_id]/[contractAddress]/settings/components/royalties.tsx

```diff
@@ -46,7 +46,7 @@ const CommonRoyaltySchema = z.object({
    * @internal
    * @remarks used by OpenSea "seller_fee_basis_points"
    */
-  seller_fee_basis_points: BasisPointsSchema.default(0),
+  seller_fee_basis_points: BasisPointsSchema,
 
   /**
    * The address of the royalty recipient. All royalties will be sent
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line +57]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->remove default from here as well<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/5342?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->

Reply:
removed